NGitLab.Tests/SnippetsTest.cs

```diff
@@ -73,5 +74,20 @@ public async Task Test_snippet_inProject(VisibilityLevel visibility)
 
             snippetClient.Delete(newSnippet.ProjectId, returnedProjectSnippet.Id);
         }
+
+        [Test]
+        public void Test_snippet_deserialize()
```

Comment:
I would expect a test that uses `snippetClient.Create`/`snippetClient.Get` or another method of `snippetClient` that returns a list of files. You can assert the list is not empty and contains the expected files.